noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/public_base_rollup.nr

```diff
@@ -204,6 +204,9 @@ mod tests {
                 FixtureBuilder::new().in_vk_tree(PUBLIC_TUBE_VK_INDEX).use_last_archive();
             inputs.avm_data = FixtureBuilder::new().in_vk_tree(AVM_VK_INDEX);
 
+            // Set the include by timestamp to the current block timestamp.
```

Comment:
```suggestion
            // Set the include by timestamp to the current block timestamp.
            // Setting it to the current block timestamp makes it essentially just a placeholder value as at this point
            // we have a guarantee of the tx being included in the block and we just need the value populated.
```
Please check that this is correct before committing as it was not super clear to me.